guidelines/index.html

```diff
@@ -1403,30 +1403,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </section>
 
             
-             <section class="guideline" id="focus-not-obscured-minimum"><div class="header-wrapper"><h4 id="x2-4-11-focus-not-obscured-minimum"><bdi class="secno">Success Criterion 2.4.11 </bdi>Focus Not Obscured (Minimum)</h4><a class="self-link" href="#focus-not-obscured-minimum" aria-label="Permalink for Section 2.4.11"></a></div>
+             <section class="guideline" id="focus-not-obscured-minimum"><div class="header-wrapper"><h4 id="x2-4-11-focus-not-obscured-minimum"><bdi class="secno">達成基準 2.4.11 </bdi>隠されないフォーカス (最低限)</h4><a class="self-link" href="#focus-not-obscured-minimum" aria-label="Permalink for Section 2.4.11"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html">Understanding Focus Not Obscured (Minimum)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#focus-not-obscured-minimum">How to Meet Focus Not Obscured (Minimum)</a></div><p class="conformance-level">(レベル AA)</p>
 	<p class="change">[New]</p>
     
-    <p>When a <a href="#dfn-user-interface-components" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-interface-components-7" title="コンテンツの一部分で、特定の機能を実現するための単一のコントロールとして利用者が知覚するもの。">user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created content.</p>
+    <p><a href="#dfn-user-interface-components" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-interface-components-7" title="コンテンツの一部分で、特定の機能を実現するための単一のコントロールとして利用者が知覚するもの。">ユーザインタフェース コンポーネント</a>がキーボードフォーカスを受け取るとき、コンテンツ制作者が作成したコンテンツによって、そのコンポーネントが完全には隠されない。</p>
 
-    <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記 1</span></div><p class="">Where content in a configurable interface can be repositioned by the user, then only the initial positions of user-movable content are considered for testing and conformance of this Success Criterion.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記 1</span></div><p class="">設定可能なインタフェースの中にあるコンテンツが利用者によって再配置可能な場合、利用者によって移動できるコンテンツの初期位置のみが、この達成基準のテスト及び適合性の対象として考慮に入れられる。</p></div>
```

Comment:
初期位置についてだけ、達成基準を満たすかどうかを検討する、というような意味かと思いますので。

```suggestion
    <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記 1</span></div><p class="">設定可能なインタフェースの中にあるコンテンツが利用者によって再配置可能な場合、利用者によって移動可能なコンテンツの初期位置だけが、この達成基準のテスト及び適合性の対象として検討される。</p></div>
```

Reply:
個人的には、@bakera さんの修正提案が、LGTM です。

Reply:
@bakera さん案のほうで、commit しようと思います。